README.md

```diff
@@ -8,6 +8,34 @@ To do this run `make run-validator` to run the validator and then run `make vali
 
```

Comment:
Should we add a mention about `test-translation-templates` from MAKEFILE?

Reply:
Added now 👍